models/Student.ts

```diff
@@ -20,11 +20,13 @@ export type CreateStudentInput = {
     therapy?: therapy[];
     otherTherapy?: string;
     guardianExpectations?: string;
+    hasMedication?: boolean;
+    hasAllergies?: boolean;
     medication?: string;
     allergies?: string;
     additionalInfo?: string;
-    emergFirstName: string;
-    emergLastName: string;
-    emergNumber: string;
-    emergRelationToStudent: string;
+    emergFirstName?: string;
+    emergLastName?: string;
+    emergNumber?: string;
+    emergRelationToStudent?: string;
```

Comment:
Looking at our schema, let's just make these fields required for now. So please make the corresponding validation in the front end!
```prisma
model Student {
  id                     Int            @id @default(autoincrement())
  parentId               Int            @map("parent_id")
  firstName              String         @map("first_name")
  lastName               String         @map("last_name")
  dateOfBirth            DateTime       @map("date_of_birth") @db.Timestamptz(6)
  addressLine1           String         @map("address_line1")
  addressLine2           String?        @map("address_line2")
  postalCode             String?        @map("postal_code") @db.VarChar(10)
  cityName               String?        @map("city_name")
  province               province?
  school                 String?
  grade                  Int?
  difficulties           difficulties[]
  otherDifficulties      String?        @map("other_difficulties")
  therapy                therapy[]
  otherTherapy           String?        @map("other_therapy")
  specialEducation       Boolean?       @default(false) @map("special_education")
  guardianExpectations   String?        @map("guardian_expectations")
  medication             String?
  allergies              String?
  additionalInfo         String?        @map("additional_info")
  emergFirstName         String         @map("emerg_first_name")
  emergLastName          String         @map("emerg_last_name")
  emergNumber            String         @map("emerg_number") @db.VarChar(50)
  emergRelationToStudent String         @map("emerg_relation_to_student")
  createdAt              DateTime       @default(now()) @map("created_at") @db.Timestamptz(6)
  updatedAt              DateTime?      @map("updated_at") @db.Timestamptz(6)
  parent                 Parent         @relation("parentsTostudents", fields: [parentId], references: [id])
  parentRegs             ParentReg[]

  @@map("students")
}
```

```suggestion
    emergFirstName: string;
    emergLastName: string;
    emergNumber: string;
    emergRelationToStudent: string;
```